scripts/asm-installer/install_asm

```diff
@@ -728,6 +733,47 @@ validate_istio_version() {
   fi
 }
 
+validate_asm_version() {
+  info "Checking existing ASM version(s)..."
+  local OUTPUT
+  OUTPUT="$(retry 3 ./istio-${RELEASE}/bin/istioctl version -o json)"
+  local FOUND_INVALID_VERSION
+  FOUND_INVALID_VERSION="false"
```

Comment:
Other places use 0 for false and 1 for true

Reply:
Currently, it's mixed style. I was following the existing **validate_istio_version** as a reference but I will updated both to match this.

Reply:
I think we should switch this. I much prefer using -eq and -ne to compare things rather than strings in bash.